lectures/python_essentials.md

```diff
@@ -61,6 +61,9 @@ Let's learn a bit more about them.
 
 ### Primitive Data Types
 
+
+***1. Boolean Values***
```

Comment:
@HumphreyYang , please use markdown syntax for headings.  This matters for the latex/pdf version.

This should be 

`#### Boolean Values`